ui/lib/core/addon/templates/components/form-field.hbs

```diff
@@ -223,6 +241,7 @@
           theme=(or attr.options.theme 'hashi')
         }}
       />
+      <p class="sub-text">{{ attr.options.helpText }}</p>
```

Comment:
We should only show this if `attr.options.helpText` exists. This also might error out if there are no options passed, so I would double check that

Reply:
True, will add defensive checks.